common/supplemental/supplementalData.xml

```diff
@@ -78,6 +78,7 @@ For terms of use, see https://www.unicode.org/copyright.html
             <info iso4217="VND" digits="0" rounding="0"/>
             <info iso4217="VUV" digits="0" rounding="0"/>
             <info iso4217="XAF" digits="0" rounding="0"/>
+			<info iso4217="XAU" digits="2" rounding="0"/>
```

Comment:
tabs versus spaces? Do we have any normalization for xml?

Reply:
probably should have a normalizer for it.